connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -200,48 +235,68 @@ public boolean isSinkConnector(String connName) {
         return result;
     }
 
-    public void stopConnectors() {
-        stopConnectors(new HashSet<>(connectors.keySet()));
+    private void stopConnectors() {
+        stopConnectors(new ArrayList<>(connectors.keySet()));
     }
 
-    public Collection<String> stopConnectors(Collection<String> connectors) {
-        final List<String> stopped = new ArrayList<>(connectors.size());
-        for (String connector: connectors) {
-            if (stopConnector(connector)) {
-                stopped.add(connector);
-            }
-        }
-        return stopped;
+    private void stopConnectors(Collection<String> connectors) {
```

Comment:
Looks like the only usage of this method is `stopConnectors()` above. Perhaps we can could just move the body into that method? I think iteration over the concurrent hash map is also safe, by the way.